rskj-core/src/test/java/co/rsk/peg/federation/FederationSupportImplTest.java

```diff
@@ -2319,6 +2319,36 @@ void voteFederationChange_commitEmptyFederation_returnsInsufficientMembersRespon
 
         }
 
+        @Test
+        void voteFederationChange_addFederatorPublicKeyWithLessThanMofNVotes_returnsPendingFederationSizeZeroResponseCode() {
```

Comment:
`_returnsPendingFederationSizeZeroResponseCode` this is not an existing response code. It is returning `SUCCESSFUL` one right?

Reply:
Updated.